src/org/usfirst/frc/team2706/robot/commands/IntakeCube.java

```diff
@@ -25,7 +25,8 @@ public void initialize() {}
      * Turns the motors on to suck in the cube
      */
     public void execute() {
-        inhale.inhaleCube();
+        inhale.inhaleCube(Robot.oi.getDriverJoystick().getRawAxis(3)); 
```

Comment:
Maybe something to look at later, but this is hardcoded, so changing the axis in Oi won't change this value.